commands/command_track.go

```diff
@@ -337,5 +369,6 @@ func init() {
 		cmd.Flags().BoolVarP(&trackNoModifyAttrsFlag, "no-modify-attrs", "", false, "skip modifying .gitattributes file")
 		cmd.Flags().BoolVarP(&trackNoExcludedFlag, "no-excluded", "", false, "skip listing excluded paths")
 		cmd.Flags().BoolVarP(&trackFilenameFlag, "filename", "", false, "treat this pattern as a literal filename")
+		cmd.Flags().BoolVarP(&trackJSONFlag, "json", "", false, "print output in json")
```

Comment:
Very minor nit since these strings are unused ... maybe "print output in JSON" here?

Reply:
Yup.  That was a copy-paste problem.